### DISCUSSION TITLE
An option that adds an explanation of the command and/or arguments

### DISCUSSION THREAD

**Discussion by joeyda3rd** [TheR1D/shell_gpt#118] [Ideas]
I'm weary to trust every command that is output after seeing the inaccuracies possible with gpt-3.5, especially if I don't fully understand all that the command and arguments do, and especially if I have never used them before. I'll just research the command before execution if I'm uncertain, but it would be helpful if the output could include a description of the command and/or arguments. Would that then be creating trust that maybe shouldn't?

**Comment by TheR1D** [comment]
This could be a useful feature that could be implemented in upcoming updates.

**Reply by joeyda3rd** [reply]
Great! I reviewed the source, it shouldn't be too hard to implement either. Thanks for considering!

**Reply by endolith** [reply]
And maybe tell it to try to use more verbose command line options instead of the single character shortcuts?  Like `rmlint --progress` instead of `rmlint -g`.

**Comment by will-wright-eng** [comment]
I've always used the [explainshell](https://github.com/idank/explainshell) project for this purpose... it looks like it's been implemented as a command line interface [here](https://github.com/7Rocky/explainsh/tree/main)
